### PR TITLE
Permission error

### PR DESCRIPTION
**Pull request by JFleming4**

### CODE REVIEW

**Comment by irushavm** [comment]
Need to rebase @JFleming4

**Comment by JFleming4** [comment]
Need Review pls

**Comment by JFleming4** [comment]
@irushavm pls

File: src/main/java/parsing/ReadCommand.java [hunk 1]
```diff
@@ -26,6 +26,7 @@ public State execute() {
 			LOG.logQuiet("Error: Not enough arguments");
 		else
 			try {
+
```
Comment (irushavm) [inline src/main/java/parsing/ReadCommand.java]: nice.

File: src/main/java/states/ReadState.java [hunk 2]
```diff
@@ -89,7 +96,16 @@ public State execute() {
 						break;
                     }
 
-                } catch (InvalidPacketException e) {
+                } catch(IOException ioE) {
+                	if(ioE.getMessage().contains("Not enough usable space")) {
+        				LOG.logQuiet("Not enough usable disk space");
+                		ErrorMessage msg = new ErrorMessage(ErrorType.DISK_FULL, "Not enough free space on disk");
+        				socket.sendMessage(msg, recvSocketAddress);
```
Comment (DerekStride) [inline src/main/java/states/ReadState.java]: This is the client, are we supposed to be sending packets like this to the server?

**Pull request merged**